apps/condo/domains/analytics/access/ExternalReport.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generated by `createschema analytics.ExternalReport 'type:Select:metabase; title:Text; description?:Text; meta?:Json'`
+ */
+
+const { throwAuthenticationError } = require('@condo/keystone/apolloErrorFormatter')
+
+async function canReadExternalReports ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return {}
+
+    return true
```

Comment:
Your case is similar to `ContactRole`, use their access. It's no OK that you can see all reports from different organizations

Reply:
Good idea, done this